H2MLauncher.UI/MainWindow.xaml

```diff
@@ -195,7 +198,7 @@
                                 </DataGrid.ItemContainerStyle>
 
                                 <DataGrid.Columns >
-                                    <DataGridTemplateColumn Header="Hostname" Width="500" MinWidth="200" MaxWidth="700">
+                                    <DataGridTemplateColumn Header="Hostname" Width="543" MinWidth="200" MaxWidth="750">
```

Comment:
Just so you are aware when merging: there are like 3 different PRs with changes to the column widths. In the end we should make it fill the width (*) but lets adjust this when everything is merged (maybe in #66 )